app/config/services.yml

```diff
@@ -1,6 +1,6 @@
 services:
     use_case.register_user:
-        class: AppBundle\UseCase\RegisterUserUseCase
+        class: AppBundle\UseCase\RegisterUser
```

Comment:
already in UseCase namespace, so suffix is no needed IMO

Reply:
Ok.